generated/nidigital/session.py

```diff
@@ -896,7 +896,7 @@ def frequency_counter_measure_frequency(self):
         frequencies_ctype = get_ctypes_pointer_for_buffer(library_type=_visatype.ViReal64, size=frequencies_size)  # case B620
         error_code = self._library.niDigital_FrequencyCounter_MeasureFrequency(vi_ctype, channel_list_ctype, frequencies_buffer_size_ctype, frequencies_ctype, None if actual_num_frequencies_ctype is None else (ctypes.pointer(actual_num_frequencies_ctype)))
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
-        return [float(frequencies_ctype[i]) for i in range(frequencies_buffer_size_ctype.value)], int(actual_num_frequencies_ctype.value)
+        return [float(frequencies_ctype[i]) for i in range(frequencies_buffer_size_ctype.value)]
```

Comment:
This function returns a list of frequencies.
The documentation above says it returns `actual_num_frequencies (int):` (also, why the trailing `:`?)

Reply:
Fixed

Also fixed: documentation for ivi-dance return values